playbooks/update-nginx-admin-password.yml

```diff
@@ -0,0 +1,23 @@
+---
+- hosts: role=control
+  serial: 1
+  tasks:
+    - name: encrypt admin password
+      sudo: yes
+      run_once: yes
+      shell: htpasswd -Bnb admin {{ nginx_admin_password | quote }} | cut -f 2 -d ':'
+      register: nginx_admin_password_encrypted
+      changed_when: no
+
+    - name: set admin password variable
+      run_once: yes
+      set_fact:
+        nginx_admin_password_encrypted: "{{ nginx_admin_password_encrypted.stdout }}"
+
+    - name: install nginx admin password
+      sudo: yes
+      run_once: yes
+      command: consul-cli kv-write service/nginx/auth/users/admin '{{ nginx_admin_password_encrypted }}'
```

Comment:
Looks like this playbook relies on the consul CLI. Could we just use `curl` + the HTTP API?